server/utils/referralUtils.ts

```diff
@@ -145,22 +145,25 @@ export default class ReferralUtils {
   static viewReferralNavigationItems(
     currentPath: Request['path'],
     referralId: Referral['id'],
+    isAssess?: boolean,
   ): Array<MojFrontendSideNavigationItem> {
+    const paths = isAssess ? assessPaths : referPaths
```

Comment:
Given we already have `currentPath` in here, could it just use `startsWith` like elsewhere instead of taking another argument? That might make it a bit less of a Refer as default arrangement